requirements/edx/testing.txt

```diff
@@ -253,6 +253,7 @@ dill==0.3.6
     # via pylint
 distlib==0.3.6
     # via virtualenv
+django==3.2.16
     # via
```

Comment:
Ha, kind of funny that we were ripping out the dependency but leaving the comments for it.